File: v2/fixtures/code-smells-go-excluded/src/deep.go
package main

func outer(a, b, c, d, e bool) string {
	if a {
		if b {
			if c {
				if d {
					if e {
						return "too deep, but .go is excluded from S-deep-nesting"
					}
				}
			}
		}
	}
	return ""
}

File: v2/fixtures/code-smells-deep-nesting/src/deep.js
function outer(a, b, c, d, e) {
  if (a) {
    if (b) {
      if (c) {
        if (d) {
          if (e) {
            return 'too deep';
          }
        }
      }
    }
  }
  return null;
}

module.exports = { outer };
